ansible/group_vars/all

```diff
@@ -0,0 +1 @@
+ansible_python_interpreter: "/usr/bin/python3"
```

Comment:
Curious what this is for...

Reply:
On some Debian systems this is needed or Ansible won't run because it can't find python.